Sources/xcodeproj/Objects/Files/PBXFileElement.swift

```diff
@@ -59,6 +62,7 @@ public class PBXFileElement: PBXContainerItem, PlistSerializable {
         self.tabWidth = tabWidth
         self.wrapsLines = wrapsLines
         super.init()
+        assignParentForChildren()
```

Comment:
I'd add a test to make sure the children get the parent assigned when the group is initialised.

Reply:
Ready